README.md

```diff
@@ -615,9 +636,10 @@ Useful links
 
 License
 -------
+This FDW linensed under PostgreSQL-like license.
```

Comment:
Please revert `License` section.

Reply:
Reverted in https://github.com/pgspider/sqlite_fdw/pull/88/commits/b78b5ed375265564fba2bb9d07d71669f89f8d32

Reply:
Could you revert Copyright notice also?

Reply:
Reverted in https://github.com/pgspider/sqlite_fdw/pull/88/commits/e81840af1156fc10c8a427591a3f5718bd42519c , please check again

Reply:
Thank you. Please help me to resolve this thread.